jac/jaclang/tests/fixtures/nested_impls.jac

```diff
@@ -0,0 +1,55 @@
+obj mainObject;
+
+:obj:mainObject{
+    can greet;
+    obj subObject;
+}
+can foo{
+     obj kk;
+     return kk();
```

Comment:
I'd like the class, function and variables to clearely state what we're testing, so in this case:

```
obj ObjInsideFunction;
```

instead of `kk`